docs/develop/terra-js/getting-started.mdx

```diff
@@ -94,14 +92,14 @@ You will also need to change the `chainID` from `"pisco-1"` to `”localterra”
 
 1. You'll need a wallet to sign and submit transactions. [Create a new wallet](../../learn/terra-station/Download/terra-station-extension.mdx) using the Terra Station extension. Be sure to save your mnemonic key!
 
-2. After creating your wallet, you’ll need to set it to use the testnet. Click the gear icon in the extension and change the network from `mainnet` to `testnet`.
+2. After creating your wallet, you will need to set it to use the testnet. Click the gear icon in the extension and change the network from `mainnet` to `testnet`.
```

Comment:
Our style guide says ccontractions are generally ok. I'll leave "you'll" to your discretion.